pykern/pkcli/github.py

```diff
@@ -579,7 +585,7 @@ def _cfg():
         test_mode=(
             pkconfig.in_dev_mode(),
             pkconfig.parse_bool,
-            f"only backs up {_TEST_REPO} repo",
+            f"only backs up {_TEST_REPOS[0]} repo",
```

Comment:
I don't think this is true. In test_mode we back up all repos in _TEST_REPOS

Reply:
That is correct, `_TEST_REPOS[0]` is the only wiki repo we back up, irrelevant for this msg.